src/sql/src/statements.rs

```diff
@@ -42,6 +42,8 @@ use crate::error::{
     SerializeColumnDefaultConstraintSnafu, UnsupportedDefaultValueSnafu,
 };
 
+// TODO(LFC): Get rid of this function, use session context aware version of "table_idents_to_full_name" instead.
```

Comment:
Why not refacor it at this time?

Reply:
some other places that are planned to be changed in the following PRs are still using it